components/esp_lvgl_port/src/lvgl9/esp_lvgl_port.c

```diff
@@ -169,23 +170,30 @@ void lvgl_port_unlock(void)
 
 esp_err_t lvgl_port_task_wake(lvgl_port_event_type_t event, void *param)
```

Comment:
`param` is not used anymore?

Reply:
Yes, in this solution cannot be used param.